kraken-app/kraken-app-controller/src/main/resources/assets/product.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     - classpath:/mef-sonata/api-specs/api-spec.quote.yaml
     - classpath:/mef-sonata/api-specs/api-spec.product.offering.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.uni.add.yaml
-    - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yam
+    - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.uni.add.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml
```

Comment:
classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml   duplicate content, would u can  delete  one of them

Reply:
Thanks. The duplicated lines have been deleted.